### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by Gallardot**
### What problem does this PR solve? 

#551 , Force the developer to `make check` before create a PR

### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -164,7 +164,7 @@ vet:
 tidy:
 	@echo "go mod tidy"
 	GO111MODULE=on go mod tidy
-	git diff --quiet go.mod go.sum
+	git diff --quiet go.mod go.sum api config controllers manifests pkg test
```
Comment (yeya24) [inline Makefile:167]: I don't think this is the change I want to see. `tidy`should only check the go module dependency.

Something like https://github.com/pingcap/tidb-operator/blob/master/hack/verify-all.sh is better

Comment (Gallardot) [reply Makefile]: Done.  PTAL

**Pull request merged**